.github/workflows/ci.yml

```
@@ -706,7 +710,18 @@ jobs:
             -Dtesting.gcp-credentials-key="${GCP_CREDENTIALS_KEY}" \
             -Dtesting.azure-abfs-container="${ABFS_CONTAINER}" \
             -Dtesting.azure-abfs-account="${ABFS_ACCOUNT}" \
-            -Dtesting.azure-abfs-access-key="${ABFS_ACCESS_KEY}"
+            -Dtesting.azure-abfs-access-key="${ABFS_ACCESS_KEY}" \
+            -Dtesting.snowflake.catalog.user=test \
+            -Dtesting.snowflake.catalog.password="${SNOWFLAKE_PASSWORD}" \
+            -Dtesting.snowflake.catalog.account="${SNOWFLAKE_ACCOUNT_NAME}" \
+            -Dtesting.snowflake.catalog.database=test \
+            -Dtesting.snowflake.catalog.schema=schema \
+            -Dtesting.snowflake.catalog.role=test \
+            -Dtesting.snowflake.catalog.warehouse=test \
+            -Dtesting.snowflake.catalog.s3.access.key="${SNOWFLAKE_CATALOG_S3_ACCESS_KEY}" \
+            -Dtesting.snowflake.catalog.s3.secret="${SNOWFLAKE_CATALOG_S3_SECRET_ACCESS_KEY}" \
```

Make sure to update the property names in the tests as well

```suggestion
            -Dtesting.snowflake.catalog.s3.access-key="${SNOWFLAKE_CATALOG_S3_ACCESS_KEY_ID}" \
            -Dtesting.snowflake.catalog.s3.secret-key="${SNOWFLAKE_CATALOG_S3_SECRET_ACCESS_KEY}" \
```